src/tests/constants.ts

```diff
@@ -1,28 +1,69 @@
+import { ChainID } from '@harmony-js/utils'
 import { ITransactionOptions } from 'src/interfaces'
+import { HarmonyShards } from '../interfaces'
+import { PrivateKey } from '../private-key'
+import 'dotenv/config'
 
 export const HARMONY_TESTNET = 'https://api.s0.b.hmny.io/'
-export const TEST_ADDRESS_1 = '0x7c61c8f282aeaae3e0476d0533d21dc42c5aa146'.toLowerCase()
-export const TEST_ADDRESS_2 = '0xc57DE96E7ADd6D0fc6BdF51758Fd90b111f89b79'.toLowerCase()
-export const TEST_ADDRESS_3 = '0x264F47C0E2fb1B587F6fa270862F3CBc68460B75'.toLowerCase()
-export const TEST_ADDRESS_4 = '0xe8f887475466f6a232f428f038b9ada36dc3582d'.toLowerCase()
+
+const TEST_PK_1 = process.env.TEST_PK_1 ?? ''
+
+if (!TEST_PK_1) {
+  throw new Error('TEST_PK_1 must have a value')
+}
+
+const TEST_SEED_1 = process.env.TEST_SEED_1 ?? ''
+
+if (!TEST_SEED_1) {
+  throw new Error('TEST_SEED_1 must have a value')
+}
+
+export const PROVIDER_TEST_1: PrivateKey = new PrivateKey(
+  HarmonyShards.SHARD_0_TESTNET,
+  TEST_PK_1.toLowerCase(),
+  ChainID.HmyTestnet,
+)
 
 export const TEST_ACCOUNT_1 = {
-  address: TEST_ADDRESS_1,
-  privateKey: 'dc0ab9fc02cb694853c939037c986a792bdd779fe481c0767d7551ad45d45b39'.toLowerCase(),
+  address: PROVIDER_TEST_1.accounts[0].toLowerCase(),
```

Comment:
Just export an address after the PrivateKey instance, like:
```ts
...
export const ADDRESS_TEST_1 = WALLET_PROVIDER_TEST_1.accounts[0].toLowerCase()
export const ADDRESS_TEST_2 = WALLET_PROVIDER_TEST_2.accounts[0].toLowerCase()
// Wallet created with the seed.
export const ADDRESS_TEST_3 = WALLET_PROVIDER_TEST_3.accounts[0].toLowerCase() 
...
```

Reply:
Sure!